src/scripts/core/cartesian.js

```diff
@@ -331,9 +333,27 @@ var cartesian = function () {
         },
 
         renderXAxis: function () {
-            var xAxis = this.xAxis();
-            var y = this.options.chart.plotHeight + this.options.chart.padding.top;
-            var x = this.options.chart.internalPadding.left;
+            const xAxis = this.xAxis();
+            const plotHeight = this.options.chart.plotHeight;
+            const paddingTop = this.options.chart.internalPadding.top;
+            const x = this.options.chart.internalPadding.left;
+            const animOpt = this.options.chart.animations;
```

Comment:
This won't work if `enable` is a function right? i.e., doesn't this need to be 'materialized'?

Reply:
`this.options` is the already materialized version of the options for the current render cycle... this.rawOptions is the original one.